vnc.sh

```diff
@@ -14,3 +14,4 @@ if command -v vncviewer >/dev/null 2>&1; then
 fi
 
 echo export BROWSER_DISPLAY=$display
+echo export DISPLAY=$display
```

Comment:
We use different variable name for a reason. So you can set `BROWSER_DISPLAY` in terminal you work in and only browser windows will be launched in vnc session.

Reply:
Ah, I'll make a comment to that effect and revert this change